include/zim/archive.h

```diff
@@ -477,6 +477,14 @@ namespace zim
       cluster_index_type getClusterCount() const;
       offset_type getClusterOffset(cluster_index_type idx) const;
       entry_index_type getMainEntryIndex() const;
+      /** Get an entry using a path and a namespace.
```

Comment:
An empty line separating the docstring from the preceding function declaration would be nice.